example/android/app/build.gradle

```diff
@@ -283,6 +283,24 @@ dependencies {
         }
     }
     // END TODO
+
```

Comment:
These dependencies were conflicting with versions in `react-native-screens`. Should be able to remove after updating `react-native-screens` version

Reply:
This really feels like something we shouldn't do. Is there another way around this?

Reply:
Can we do the `react-native-screens` update before this PR?

Reply:
Yeah I'm looking into this now. Based on [this](https://github.com/software-mansion/react-native-screens) we might have to upgrade the react native version to upgrade the react-native-screens dep to a version that doesn't cause conflicts. Alternatively, I could just update stripe-android to 20.52.0 and then we can handle the dependency conflicts for the major version update

Reply:
Downgraded to 20.52.0